src/client/components/DataCard.js

```diff
@@ -9,4 +10,9 @@ function DataCard({ title, data }) {
   );
 }
 
+DataCard.propTypes = {
+  title: PropTypes.string,
+  data: PropTypes.string
```

Comment:
I am not sure if we are using this only for string. It can be number or decimal. Please check with Zaki.

Reply:
Actually it should be a string. But that means that `CostWidget` needs to pass it down as a string not as a number, so we might need to change that component. Or add different classes of DataCards. Let's make it a string for now and see where it takes us.